docs/platform/concepts/free-trial.rst

```diff
@@ -0,0 +1,34 @@
+30-day trials
+==============
+
+Aiven offers a free trial for 30 days to explore the platform and the `Aiven services <https://docs.aiven.io/>`_. You don't need a credit card to sign up.
+
+If you create services on the :doc:`free plan </docs/platform/concepts/free-plan>`, they will continue to run alongside your free 30-day trial without affecting your trial credits. Free plan services also continue running after the trial has expired.
+
+Trial features and limitations
+-------------------------------
+
+The 30-day trial gives you full access to the `Aiven Console <https://console.aiven.io/>`_, :doc:`API </docs/tools/api>`, :doc:`CLI </docs/tools/cli>`, :doc:`Aiven Terraform provider </docs/tools/terraform>`, and :doc:`Aiven Operator for Kubernetes </docs/tools/kubernetes>`. The credits can be used on any service types and plan sizes. 
+
+Trials include:
+
+* Up to 10 VMs
+
+* 1 :doc:`Virtual Private Cloud (VPC) </docs/platform/howto/manage-vpc-peering>`
+
+* Up to 10 VPC peering connections
+
+
+There are some limitations:
+
+* Trial projects are automatically created and cannot be renamed
```

Comment:
Technically you can rename the project during signup, but at that point it isn't a trial project yet. Maybe we could rephrase this to: "You can only create a trial in your default project, i.e. the first project that is created when you sign up."